### PR TITLE
add haskell-links

### PR DESCRIPTION
**Pull request by simonmichael**

### CODE REVIEW

**Comment by reubenharry** [comment]
Thanks for this! I left one comment.

**Comment by simonmichael** [comment]
I'm not seeing your comment ?

**Comment by reubenharry** [comment]
Oops. There now!

File: docs/docs/resources/resources.md [hunk 1]
```diff
@@ -22,8 +22,11 @@ It also has a [forum](https://discourse.haskell.org/).
 
 ## Understanding the ecosystem
 
+[haskell-links.org](https://haskell-links.org) presents the links saved by #haskell IRC/Matrix chatters, with a fast search UI, redirect urls, and light curation.
```
[the remark this answers is not in the document]
Comment (simonmichael) [reply docs/docs/resources/resources.md]: I have trimmed the description as you suggest. I left it as first link as it (the About section) seems a better first high-level overview of the available resources (including the SOTU doc). If you disagree, feel free to change it and merge. Thanks!

**Pull request merged**